miio/vacuum_cli.py

```diff
@@ -145,6 +145,29 @@ def consumables(vac: miio.Vacuum):
 
 
 @cli.command()
+@click.argument('name', type=str, required=True)
+@pass_dev
+def reset_consumable(vac: miio.Vacuum, name):
+    """Reset consumable."""
+    from miio.vacuum import Consumable
+    if name == 'main_brush':
+        consumable = Consumable.MainBrush
+    elif name == 'side_brush':
+        consumable = Consumable.SideBrush
+    elif name == 'filter':
+        consumable = Consumable.Filter
+    elif name == 'sensor_dirty':
+        consumable = Consumable.SensorDirty
+    else:
+        return
```

Comment:
I think it makes sense to error out here before returning, if not just to avoid bug reports caused by potential misspellings / name changes.